### ISSUE TITLE
 Drupal\Component\Plugin\Exception\PluginNotFoundException: The "" entity type does not exist. 

### ISSUE THREAD

**Issue by finnlewis** [localgovdrupal/localgov_services#87]
On a fresh install, if one creates a services sub landing page with a link to front page with either <front> or / the page will not render. 

I think this might be because no home page is defined out of the box.  

```
The website encountered an unexpected error. Please try again later.

Drupal\Component\Plugin\Exception\PluginNotFoundException: The "" entity type does not exist. in Drupal\Core\Entity\EntityTypeManager->getDefinition() (line 150 of core/lib/Drupal/Core/Entity/EntityTypeManager.php). 
```

### Steps to reproduce:

Fresh install of the localgov profile.
Create a service landing page https://localgov.lndo.site/node/add/localgov_services_landing 
Create a service sublanding page https://localgov.lndo.site/node/add/localgov_services_sublanding
Add a reference to the landing page. 
Add a child link with <front> or / in the link field.
Save.
Try to view the page.

**Comment by keelanfh** [comment]
@finnlewis is this one closed by #191 as well - you might want to close it?

**Comment by finnlewis** [comment]
Good call, thanks @keelanfh !
